app.js

```diff
@@ -87,41 +89,51 @@ app.use(function(req, res, next) {
 });
 
 /**
+ * Is this a static asset?
+ */
+app.use('/', express.static(DIST_DIR));
```

Comment:
this relocation is what's causing the 303 error in app.test.js, because `express.static` does not have graceful `next()` fallthrough